crates/viewer/re_renderer_examples/world_grid.rs

```diff
@@ -0,0 +1,134 @@
+//! Demonstrates world grid rendering.
+//!
+//! Controls:
+//! - Space: Pause
+//! - G: Toggle camera mode
```

Comment:
It would be really useful to control the camera; at least the angle of it

Reply:
#1426